extension-manifest-v2/src/utils/utils.js

```diff
@@ -229,60 +227,25 @@ export function getActiveTab(callback, error) {
 	});
 }
 
-/**
- * Helper called by openNewTab.
- * @memberOf BackgroundUtils
- * @private
- *
- * @param  {Object} data 	info about the new tab
- */
-function _openNewTab(data) {
-	getActiveTab((tab) => {
-		if (tab) {
-			chrome.tabs.create({
-				url: data.url,
-				active: data.become_active || false,
-				openerTabId: tab.id,
-				index: tab.index + 1
-			});
-		} else {
-			chrome.tabs.create({
-				url: data.url,
-				active: data.become_active || false
-			});
-		}
-	}, (err) => {
-		log(`_openNewTab Error: ${err}`);
+export async function openNewTab({ url, become_active: active = false } = {}) {
+	if (!url) { return; }
```

Comment:
Is it being reach? Or would it be better to let it throw an error if no URL is given?